src/main/java/img2md/DirCache.java

```diff
@@ -0,0 +1,65 @@
+package img2md;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DirCache {
+
+    private static volatile List<String> dirs = new ArrayList<>();
+
+    public static String[] get() {
+        if (dirs.isEmpty()) {
+            try {
+                String userHome = System.getProperty("user.home");
+                String filePath = userHome + "/.pasteimages" + "/cache.txt";
```

Comment:
I would expect a per project cache or in the `.idea` directory?

Later after some time nobody knows where the data in `~/.pasteimages` comes from.

Reply:
So true, but I would first need to do some research about how to implement that. Could you provide a PR?